rotkehlchen/chain/ethereum/modules/yearn/utils.py

```diff
@@ -114,13 +111,11 @@ def query_yearn_vaults(db: 'DBHandler') -> None:
 
     assert data is not None, 'data exists. Checked by _maybe_reset_yearn_cache_timestamp'
     for vault in data:
-        if 'type' not in vault or (vault['type'] == '' and 'version' not in vault):
+        if 'version' not in vault:
             log.error(f'Could not identify the yearn vault type for {vault}. Skipping...')
             continue
 
-        if vault['type'] == 'v1':
-            vault_type = YEARN_VAULTS_V1_PROTOCOL
-        elif vault['type'] == 'v2' or vault['version'].startswith('0.'):  # version '0.x.x' happens in ydemon and is always a v2 vault  # noqa: E501
```

Comment:
why did you remove the vault['type'] check here?

Reply:
This was apparently something from the old api that was removed here: https://github.com/rotki/rotki/commit/5ec2771ebbef85b8750cad5a31eaaa90f444b197. In the data from the ydaemon API, `type` is either `Yearn Vault`, `Automated Yearn Vault`, or something similar, and can't be used to determine the vault version.